models/llama-2-13b-mlc/config.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from dotenv import load_dotenv
+from src.config_utils import Weights, get_mlc_file_list, mlc_kwargs
+from src.inference_engines.mlc_engine import MLCEngine
+from src.utils import get_env_var_or_default
+
+load_dotenv()
+
+MODEL_NAME = "llama-2-13b-mlc"
+
+# Inference weights
+mlc_file_list = get_mlc_file_list(
+    model_name="llama-2-13b-hf-q4f16_1", n_shards=163)
+
+LOCAL_PATH = f"models/{MODEL_NAME}/model_artifacts/default_inference_weights"
+
+# ensure directories exist
```

Comment:
nit: this is fine for now - it might be good to refactor it into the MLC engine instead of running code beyond function calls in `config.py`

Reply:
Highly agreed. Doing this now.

Reply:
Done!